docs/Error-Handling.md

```diff
@@ -1,56 +0,0 @@
-<h1 align="center">Fastify</h1>
-
-## Error classification in Promises & Async/Await
```

Comment:
Can be rewritten to support the current state. It contains some useful recommendations. Should I create a PR ?

Reply:
Definitely, you can push to this branch as well.